Modules/SkeletonDPL/src/SkeletonTaskDPL.cxx

```diff
@@ -0,0 +1,98 @@
+///
+/// \file   SkeletonTask.cxx
+/// \author Barthelemy von Haller
+///
+
+#include <TH1.h>
+#include <TCanvas.h>
+
+#include "SkeletonDPL/SkeletonTaskDPL.h"
+#include "QualityControl/QcInfoLogger.h"
+
+using namespace std;
+
+namespace o2 {
+namespace quality_control_modules {
+namespace skeleton_dpl {
+
+SkeletonTaskDPL::SkeletonTaskDPL()
+  : TaskInterfaceDPL(), mHistogram(nullptr)
+{
+  mHistogram = nullptr;
+}
+
+SkeletonTaskDPL::~SkeletonTaskDPL()
+{
+}
+
+void SkeletonTaskDPL::initialize(o2::framework::InitContext& ctx)
+{
+  QcInfoLogger::GetInstance() << "initialize SkeletonTaskDPL" << AliceO2::InfoLogger::InfoLogger::endm;
+
+  mHistogram = new TH1F("example", "example", 20, 0, 10000);
+  getObjectsManager()->startPublishing(mHistogram);
+//  getObjectsManager()->addCheck(mHistogram, "checkFromSkeleton",
+//                                "o2::quality_control_modules::skeleton_dpl::_SkeletonCheck",
+//                                "QcSkeletonDpl");
+}
+
+void SkeletonTaskDPL::startOfActivity(Activity &activity)
+{
+  QcInfoLogger::GetInstance() << "startOfActivity" << AliceO2::InfoLogger::InfoLogger::endm;
+  mHistogram->Reset();
+}
+
+void SkeletonTaskDPL::startOfCycle()
+{
+  QcInfoLogger::GetInstance() << "startOfCycle" << AliceO2::InfoLogger::InfoLogger::endm;
+}
+
+void SkeletonTaskDPL::monitorDataBlock(o2::framework::ProcessingContext& ctx)
+{
+  QcInfoLogger::GetInstance() << "monitorDataBlock" << AliceO2::InfoLogger::InfoLogger::endm;
+
+  // exemplary ways of accessing inputs:
```

Comment:
I fear that detectors' physicists will not know what is an input here. Maybe a word of explanation targeted at laymen could help. How does it map with the configuration people have put in the config ?